File: BH Tic Tac Toe NetBeans Project/src/org/bh/game/ttt/evt/GameStateChangeListener.java
package org.bh.game.ttt.evt;

import java.util.EventListener;

/**
 * GameStateChangeListener, made for BH Tic Tac Toe NetBeans Project, is copyright Blue Husky Programming ©2014 GPLv3 <hr/>
 * 
 * @author Kyli of Blue Husky Programming
 * @version 1.0.0
		- 2014-09-24 (1.0.0) - Kyli created GameStateChangeListener
 * @since 2014-09-24
 */
public interface GameStateChangeListener extends EventListener
{
	public void gameStateChanged(GameStateChangeEvent evt);
}

File: BH Tic Tac Toe IntelliJ Project/src/org/bh/game/ttt/evt/GameStateChangeListener.kt
package org.bh.game.ttt.evt

import java.util.*

/**
 * GameStateChangeListener, made for BH Tic Tac Toe NetBeans Project, is copyright Blue Husky Software ©2014 GPLv3 <hr/>
 *
 * @author Kyli & Ben of Blue Husky Software
 * @version 2.0.0
 *          ! 2016-10-10 (2.0.0) - Ben migrated to Kotlin
 *          . 2014-09-24 (1.0.0) - Kyli created GameStateChangeListener
 * @since 2014-09-24
 */
interface GameStateChangeListener : EventListener {
    fun gameStateChanged(evt: GameStateChangeEvent)
}
